src/syrupy/session.py

```diff
@@ -23,11 +23,9 @@
 
 @attr.s
 class SnapshotSession:
-    base_dir: str = attr.ib()
-    update_snapshots: bool = attr.ib()
-    warn_unused_snapshots: bool = attr.ib()
-    include_snapshot_details: bool = attr.ib()
-    _invocation_args: Tuple[str, ...] = attr.ib(factory=tuple)
+    # pytest.Session
+    _pytest_session: Any = attr.ib()
```

Comment:
you can reference the type directly, no need for "Any"

Reply:
It was causing issues with mypy